### PR TITLE
jquery showcases based on storybook

### PR DESCRIPTION
**Pull request by dabeng**
## Description
Port the jQuery showcase examples to storybook

Here's the related [Jira story](https://patternfly.atlassian.net/browse/PTNFLY-2567).

## Link to rawgit and/or image
Please review the [storybook](https://rawgit.com/dabeng/patternfly/jquery-showcase-dist/dist/tests/jquery/storybook/index.html) for jquery showcase site.

### CODE REVIEW

**Comment by andresgalante** [comment]
@priley86 Can you please review this one?

**Comment by priley86** [comment]
@andresgalante of course ;) 

Excited to dive into this!

**Comment by priley86** [comment]
@dabeng i seem to be getting an empty page when loading the Storybook?

**Comment by priley86** [comment]
@dabeng this seems to loading pretty slow currently (as the webpack bundle is over 6MB). We haven't seemed to run into any issues with this in React PF so I'll have to dive into your Storybook Webpack configs... this looks very promising though!

File: tests/pages/jquery/stories/index.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+import VerticalNavigationDoc from './VerticalNavigation.md';
+import Wizard from './Wizard';
+import WizardDoc from './Wizard.md';
+
+import jquery from 'jquery';
+global.$ = jquery;
+global.jQuery = jquery;
+require('bootstrap');
+require(__dirname + '/../../../../dist/js/patternfly.min.js');
+require('jquery-match-height');
```
Comment (priley86) [inline tests/pages/jquery/stories/index.js:69]: I would do all of these requires individually in each story file

File: tests/pages/jquery/stories/index.js [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+require('patternfly-bootstrap-treeview');
+require('datatables.net');
+require('datatables.net-select');
+require('eonasdan-bootstrap-datetimepicker');
+
+import '../../../../dist/css/patternfly.min.css';
+import '../../../../dist/css/patternfly-additions.min.css';
+import '../../../../dist/css/patternfly-jquery.min.css';
+
+storiesOf('Application Framework', module)
```
Comment (priley86) [inline tests/pages/jquery/stories/index.js:84]: You can split stories into separate files that can be loaded individually using config.js... example in the pf-react repo here: https://github.com/patternfly/patternfly-react/blob/master/storybook/config.js#L7

**Comment by jeff-phillips-18** [comment]
Looks like a lot of duplication of our test pages 😬

**Comment by LHinson** [comment]
@dabeng are you ok closing this one for now given my recent e-mail about setting this effort aside? 

@jeff-phillips-18 @dgutride any concerns?

**Comment by jeff-phillips-18** [comment]
@LHinson No concerns here.

**Pull request closed without merging**